cli/flox/doc/manifest.toml.md

```diff
@@ -329,11 +283,6 @@ bash = """
     alias foo="echo bar"
     set -o vi
 """
-fish = """
-    source $venv_dir/bin/activate.fish
-    alias foo="echo bar"
-    fish_vi_key_bindings
-"""
```

Comment:
blocking: bad merge

Reply:
i fixed the manpage category and fish, what else got merged badly?,

Reply:
That's it 👍